app/controllers/admin/organizations_controller.rb

```diff
@@ -7,4 +7,58 @@ def index
       per_page: 50
     )
   end
+
+  # show view includes edit
+  def show
+    @org = Organization.find(params[:id])
+    @countries = Country.all
+    @form_action_url = admin_organization_path
```

Comment:
This relates to the organizations table. Changes in this branch were to the loans table. I can address this in a separate pull request.